src/plugins/imagery/components/imageryViewLayout.vue

```diff
@@ -0,0 +1,290 @@
+                        <a class="c-button show-thumbs sm hidden icon-thumbs-strip" />
+                        <span class="l-time">{{getTime()}}</span>
+                    </div>
+                    <div class="h-local-controls flex-elem">
+                        <a class="c-button icon-pause pause-play"
+                                @click="paused(!paused())"
+                                :class="{'is-paused': paused()}" />
+                    </div>
+                </div>
+            </div>
+        </pane>
+
+        <pane class="c-inspector__elements" handle="before"
+                :style="{'min-height': `100px`}">
+            <div class="c-elements-pool">
+                <div class="thumbs-layout" ref="thumbsWrapper"
+                        @scroll="handleScroll">
+                    <div class="l-image-thumb-item"
+                            :class="{selected: image.selected}"
+                            v-for="(image, index) in imageHistory"
+                            :key="index"
+                            @click="setSelectedImage(image)">
+                        <img class="l-thumb" v-bind:src="getImageUrl(image)">
+                        <div class="l-time">{{getTime(image)}}</div>
+                    </div>
+                </div>
+            </div>
+        </pane>
+    </multipane>
+</template>
+
+<style lang="scss">
+    @import '~styles/sass-base';
+
+    .c-imagery-layout {
+        display: flex;
+        flex-direction: column;
+        overflow: auto;
+
+        .main-image-wrapper {
+            display: flex;
+            flex-direction: column;
+            height: 100%;
+            padding-bottom: 5px;
+        }
+
+        .main-image {
+            background-position: center;
+            background-repeat: no-repeat;
+            background-size: contain;
+            height: 100%;
+
+            &.unnsynced{
+                @include sUnsynced();
+            }
+        }
+
+        .l-image-controller {
+            padding: 5px 0 0 0;
+        }
+
+        .thumbs-layout {
+            margin-top: 5px;
+            overflow: auto;
+        }
+    }
+</style>
+
+<script>
+import multipane from '@/ui/layout/multipane.vue';
+import pane from '@/ui/layout/pane.vue';
+import _ from 'lodash';
+
+export default {
+    inject: ['openmct', 'domainObject'],
+    components: {
+        multipane,
+        pane,
+    },
+    data() {
+        return {
+            autoScroll: true,
+            clipped: false,
+            date: '',
+            elementHeight: this.getElSize(),
+            filters : {
+                brightness: 100,
+                contrast: 100
+            },
+            image: {
+                selected: ''
+            },
+            imageFormat: '',
+            imageHistory: [],
+            imageUrl: '',
+            isPaused: false,
+            requestCount: 0,
+            timeFormat: '',
+        }
+    },
+    methods: {
+        datumMatchesMostRecent (datum) {
```

Comment:
No space is needed between the function name and the opening parenthesis for consistency.